src/sdk/account/toNexusAccount.ts

```diff
@@ -371,7 +349,7 @@ export const toNexusAccount = async (
         module.address
       ])
 
-      const accountAddress = await getAddress()
+      const accountAddress = await getCounterFactualAddress()
```

Comment:
Can we use `getAddress()` terminology everywhere in our tests please? Makes a bit more sense for users to see that term being used instead.

Reply:
Tests and throughout our src code too where appropriate.

Reply:
We do use getAddress() everywhere still, "getCounterFactualAddress()" is used only internally in the toNexusAccount.ts